### PR TITLE
EOS-26899: Message bus server common init in utils server

### PR DESCRIPTION
**Pull request by suryakumar1024**
Signed-off-by: suryakumar.kumaravelan <suryakumar.kumaravelan@seagate.com>

# Problem Statement
- Make message bus init  call inside utils server itself so that other URLs method defined in utils routs can able to access the message config

# Design
-  For Bug describe the fix here. 
-  For Feature, Post the link to the solution page on the confluence CORTX Foundation Library 

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: 
-  Confirm All CODACY errors are resolved [Y/N]: 

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [x] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [x] Is there a change in filename/package/module or signature [Y/N]: 
- [x] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [ ] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

**Review by vastradparayya — APPROVED** [review]

File: py-utils/test/message_bus/test_message_bus_server.py [hunk 1]
```diff
@@ -29,24 +29,16 @@ class TestMessage(unittest.TestCase):
     """Test MessageBus rest server functionality."""
 
     _base_url = 'http://0.0.0.0:28300/MessageBus/message/'
-    _message_type = 'test'
+    _message_type = 'testit12'
```
Comment (selvakumaar5496) [inline py-utils/test/message_bus/test_message_bus_server.py:32]: Why message_type is changed?

File: py-utils/src/utils/utils_server/utils_server.py [hunk 2]
```diff
@@ -45,22 +56,27 @@ def __init__(self):
...
         help="Cluster config file path for Support Bundle",\
         default='yaml:///etc/cortx/cluster.conf')
     args=parser.parse_args()
-
-    CortxConf.init(cluster_conf=args.cluster_conf)
+    cluster_conf = args.cluster_conf
+    CortxConf.init(cluster_conf=cluster_conf)
     # Get the log path
-    log_dir = CortxConf.get('log_dir', '/var/log')
+    log_dir = CortxConf.get('log_dir')
```
Comment (selvakumaar5496) [inline py-utils/src/utils/utils_server/utils_server.py:69]: We need to get rid of CortxConf. 
log_path should be read from confstore

File: py-utils/test/message_bus/test_message_bus_server.py (hunk 1 above)
Comment (suryakumar1024) [reply py-utils/test/message_bus/test_message_bus_server.py]: reverted in recent patch

File: py-utils/src/utils/utils_server/utils_server.py (hunk 2 above)
Comment (suryakumar1024) [reply py-utils/src/utils/utils_server/utils_server.py]: Updated in recent patch

File: py-utils/src/utils/utils_server/utils_server.py [hunk 3]
```diff
@@ -16,20 +16,31 @@
...
+    """Base class for Cortx Rest Server implementation."""
 
-    def __init__(self):
+    def __init__(self, message_server_endpoints):
         app = web.Application()
         from cortx.utils.iem_framework import IemRequestHandler
         from cortx.utils.message_bus import MessageBusRequestHandler
         from cortx.utils.audit_log import AuditLogRequestHandler
+
+        MessageBus.init(message_server_endpoints=message_server_endpoints)
```
Comment (ujjwalpl) [inline py-utils/src/utils/utils_server/utils_server.py:43]: utils_sever is a framework and MUST not call Message Bus Init. 

Create a class called MesageServer, derive from UtilsServer and Message Bus Init should be included in that.

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

**Review by veerendra-simha-garikipati — CHANGES_REQUESTED** [review]

**Review by selvakumaar5496 — APPROVED** [review]

**Review by veerendra-simha-garikipati — APPROVED** [review]

**Comment by suryakumar1024** [comment]
Closing this PR. Raised new one for the same requirements https://github.com/Seagate/cortx-utils/pull/677

**Pull request closed without merging**